### PR TITLE
Add projects to homepage

### PR DESCRIPTION
**Pull request by chidiebereojingwa**
## Add projects to homepage

Added my projects to the homepage using
- html
- css
- linters

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -121,10 +129,608 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 0;
+  position: absolute;
+  width: 375px;
+  height: 3455px;
+  left: 0;
+  top: 682px;
+}
```
Comment (TedLivist) [inline css/style.css]: - [ ] The main container of the projects should have `display: grid`

File: css/style.css [hunk 2]
```diff
@@ -121,10 +129,608 @@
...
+  top: 107px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 24px 0;
+}
+
+.first {
+  display: flex;
+  flex-direction: column;
```
Comment (TedLivist) [inline css/style.css:170]: - [ ] You seem to have used this combination of stylings repeatedly in your CSS file. In keeping with the DRY principles, please remove these stylings into a different CSS class and then add that class as an attribute in your HTML tags.

For example:
**CSS**
```
.d-flex {
  display: flex;
}

.d-flex-col {
  flex-direction: column;
}
```
**HTML**
```
<div class="first d-flex d-flex-col">...</div>
```

File: css/style.css [hunk 3]
```diff
@@ -121,10 +129,608 @@
...
+  position: static;
+  width: 375px;
+  height: 568px;
+  margin: 16px;
+  background-image: url(../assets/projectBackground1.png);
+  background-size: initial;
+  background-attachment: initial;
+  background-position: 39px 50px;
+  background-repeat: no-repeat;
+}
```
Comment (TedLivist) [inline css/style.css]: - [ ] Your `.second`, `.third`, `.fourth`, `.fifth` and `.sixth` class stylings are almost the same. Please, use DRY principles and use one class instead. Then add that class in the different HTML elements.

File: index.html [hunk 4]
```diff
@@ -4,42 +4,175 @@
...
+
+    <div class="second">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="grid-container2">
+            <li class="stack2">html</li>
+            <li class="stack2">bootstrap</li>
+            <li class="stack2">Ruby</li>
+        </ul>
+        <a href="#" class="button">See Project</a>
```
Comment (TedLivist) [inline index.html:98]: - [ ] Your 'See Project' links should have styling for `:active`. Currently they only have styling `:hover` in your CSS.

File: index.html [hunk 5]
```diff
@@ -4,42 +4,175 @@
...
+    </div>
+
+    <div class="third">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
```
Comment (TedLivist) [inline index.html]: - [ ] Your 'HTML', 'Bootstrap, and 'Ruby' should be within a `ul` and each of them should be a `li`. Also, do the same for your other projects containers.

javas

File: index.html [hunk 6]
```diff
@@ -4,42 +4,175 @@
...
+                <h2 class="lang">Skill</h2>
+            </div>
+            <ul class="skillElement">
+             <li class="codekit">Codekit</li>
+            <li class="github">Github</li>
+            <li class="codepen">Codepen</li>
+            <li class="gitlab">Gitlab</li>
+            <li class="terminal">Terminal</li>
+            </ul>         
+
```
Comment (TedLivist) [inline index.html]: - [ ] Please correct your indentations and remove unnecessary blank lines. This makes for better code readability and organization.

File: index.html [hunk 7]
```diff
@@ -4,42 +4,175 @@
...
+        <a href="#" class="button">See Project</a>
+    </div>
 </section>
 
+<section class="item-3">
+    <div class="about">
+
+        <h2 class="aboutMe">About me</h2>
+        <p class="decription">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a class="seeProject">Get My Resume</a>
```
Comment (TedLivist) [inline index.html:167]: - [ ] Please apply the correct button background color as defined in the template

- [ ] Also, there should be a styling for the `:active` attribute in CSS as defined in the template:

btns-styles

File: index.html [hunk 8]
```diff
@@ -4,42 +4,175 @@
...
+
+        <img class="bottomline" src="assets/buttomline.png" alt="border line">
+
+        <div class="language">
+            <div class="triangle">
+                <img class="rect" src="assets/Rectangle%2055.png" alt="rentangle">
+                <h2 class="lang">Language</h2>
```
Comment (TedLivist) [inline index.html]: - [ ] Please enclose your **Language**, **Framework** and **Skills** categories inside `ul` tags with each of them in a `li` tag.

langs

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @chidiebereojingwa,

Good job so with your UI design ✔️ 
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Please use the correct colors for your 'See Project buttons' as defined in the Template:

chidi-backg

- [ ] **_Check the comments under the review._**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -4,42 +4,175 @@
...
+        <img class="firstProjectImg" src="./assets/imgplaceholder.png" alt="project Image">
+        <h2 class="multi"> Multi-Post Stories</h2>
+        <p class="story">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="grid-container">
+            <li class="stack">css</li>
+            <li class="stack">html</li>
+            <li class="stack">bootstrap</li>
+            <li class="stack">Ruby</li>
         </ul>
+        <a class="project">See Project</a>
```
Comment (Mihndim2020) [inline index.html:84]: - [ ] As requested by the former code reviewer, which is also part of the project requirements, and a best practice to stick to the client's design. Please make sure you buttons("See Project") matches those in the design. 

***The design***
chiD

***Your implementation***
chiI

File: index.html [hunk 10]
```diff
@@ -4,42 +4,175 @@
...
+
+    <div class="second pos">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="grid-container2">
+            <li class="stack2">html</li>
+            <li class="stack2">bootstrap</li>
+            <li class="stack2">Ruby</li>
+        </ul>
+        <a href="#" class="button">See Project</a>
```
Comment (Mihndim2020) [inline index.html:98]: - [x] As requested by the former code reviewer, please kindly style your buttons according to the design specifications. 

***The design***
chiBut

File: index.html [hunk 11]
```diff
@@ -4,42 +4,175 @@
...
+    </div>
+
+    <div class="third pos">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
```
Comment (Mihndim2020) [inline index.html]: - [ ] As requested by the former code reviewer, which is also part to the project requirements, the project technologies should be a list `<ul>` and each of them is a list item `<li>`

File: index.html [hunk 12]
```diff
@@ -4,42 +4,175 @@
...
+                <img class="rect" src="assets/circle.png" alt="circle">
+                <h2 class="lang">Skill</h2>
+            </div>
+            <ul class="skillElement">
+             <li class="codekit">Codekit</li>
+            <li class="github">Github</li>
+            <li class="codepen">Codepen</li>
+            <li class="gitlab">Gitlab</li>
+            <li class="terminal">Terminal</li>
+            </ul>
```
Comment (Mihndim2020) [inline index.html]: - [ ] As  requested by the former code reviewer, please make sure your code is properly indented and remove unnecessary white spaces, as this makes your code more readable.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @chidiebereojingwa, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there


 # Status: Requires Changes ♻️
_Check the comments above :point_up:  the review._


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 11 above)
Comment (chidiebereojingwa) [reply index.html]: already implemented 
image

File: index.html (hunk 7 above)
Comment (chidiebereojingwa) [reply index.html:167]: Please this is for Desktop version

File: index.html (hunk 10 above)
Comment (chidiebereojingwa) [reply index.html:98]: Please this is for Desktop version

Comment (chidiebereojingwa) [reply index.html:98]: https://www.loom.com/share/368675bdbf4244e5968490a59a68cf8a

Comment (TedLivist) [reply index.html:98]: Hey @chidiebereojingwa. You seem to have misunderstood the template. It is a requirement for the mobile (look at the image below). And by the way, the same button styling applies to both mobile and desktop.

btn-req

**Review by Alexoid1 — APPROVED** [review]
# STATUS: APPROVED!!!
## Hi @chidiebereojingwa  Great job! 
## Congratulations :tada: Thanks for implementing changes suggested.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:

# OPTIONAL
- [ ] The button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design).

brent-rambo-thumbs-up

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**